sys-apps/fwupd/fwupd-1.3.2.ebuild

```diff
@@ -14,10 +14,11 @@ SRC_URI="https://github.com/hughsie/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
 LICENSE="LGPL-2.1+"
 SLOT="0"
 KEYWORDS="~amd64 ~arm ~x86"
-IUSE="agent amt consolekit dell gtk-doc elogind +gpg introspection +man nvme pkcs7 redfish synaptics systemd test thunderbolt uefi"
+IUSE="agent amt consolekit dell gtk-doc elogind minimal +gpg introspection +man nvme pkcs7 redfish synaptics systemd test thunderbolt uefi"
 REQUIRED_USE="${PYTHON_REQUIRED_USE}
 	^^ ( consolekit elogind systemd )
```

Comment:
I think none of consolkit, elogind or systemd is required for standalone

Reply:
Oh, nice, let me modify it

Reply:
Alright, I think I fixed it. Please take another look @joakim-tjernlund

Reply:
Looks much better, thanks.
I guess that [introspection?] means that introspection is optional ?

Reply:
Correct: https://dev.gentoo.org/~ulm/pms/head/pms.html#x1-830008.3.4